practices/graded-browser-support.md

```diff
@@ -1,52 +1,48 @@
 # Graded Browser Support
 
-
 ## Our criteria for browser support
 
-
 We follow the guidelines for graded browser support outlined by Nate Koechley at Yahoo:
 [Yahoo Grade Browser Support guide](https://github.com/yui/yui3/wiki/Graded-Browser-Support)
 
-
-
 | Tables        | Grade-A support | Grade-C support  |
 | ------------- |:---------------:| ----------------:|
 | Chrome        | latest stable   |                  |
 | Edge          | latest stable   |                  |
-| Firefox       | latest stable   |               $1 |
+| Firefox       | latest stable   |               <2 |
 | IE            | 9, 10, 11       |               <9 |
 | Opera         | latest stable   |              <10 |
 | Safari        | latest stable, latest stable -1   |  <4.1 (desktop) |
 | Webkit        | Android 4.&#8224; |  <3           |
 
-
-
-
-
 ## How we implement graded browser support
 
+Our primary approach to graded browser support is the "cutting the mustard" progressive enhancement technique, based upon the principles developed by the BBC: [Cutting the Mustard](http://responsivenews.co.uk/post/18948466399/cutting-the-mustard).  However, user agent sniffing is still applied on some legacy projects.
 
-There are currently two methods by which we implement graded support for browsers using principles of progressive enhancement.
+We load a basic stylesheet to all users. This contains only [normalisation](https://necolas.github.io/normalize.css/) and a few small enhancements to the user-agent stylesheet.
+To load the full experience only in modern browsers we implement logic in the media attribute of the `<link>` element that identifies the main stylesheet, loading the stylesheet only in browsers that recognise the properties of that media query.
+This technique is documented here: [Cutting the Mustard with Media queries](https://www.sitepoint.com/cutting-the-mustard-with-css-media-queries/).
 
-In some instances we use user-agent sniffing to determine if a users browser falls into our pre-determined group of grade-c browsers, in which case we serve only the basic stylesheet and exclude the full experience.
```

Comment:
Haha oh my god yes you did do that. I even looked at it and double-checked that it worked properly because there's a double space at the beginning of the sentence and _still_ forgot about it by the time I got to line 31. 

No, it doesn't need more than that, I'm just a forgetful clown 😂